Src/WindowsServer/WindowsServer.Shared.Tests/FirstChanceExceptionStatisticsTelemetryModuleTest.cs

```diff
@@ -341,14 +453,9 @@ public void FirstChanceExceptionStatisticsTelemetryModuleDoNotIncrementOnRethrow
             Assert.Equal(1, metrics[0].Value, 15);
             Assert.Equal(0, metrics[1].Value, 15);
 
-            Assert.Equal(2, this.items.Count);
-
-            Assert.Equal(1, ((MetricTelemetry)this.items[0]).Count);
-            Assert.Equal(1, ((MetricTelemetry)this.items[1]).Count);
-
-            // One of them should be 0 as re-thorwn, another - one
-            Assert.Equal(0, Math.Min(((MetricTelemetry)this.items[0]).Sum, ((MetricTelemetry)this.items[1]).Sum), 15);
-            Assert.Equal(1, Math.Max(((MetricTelemetry)this.items[0]).Sum, ((MetricTelemetry)this.items[1]).Sum), 15);
+            Assert.Equal(1, this.items.Count);
```

Comment:
It looks like change in the dim capping also affected Rethrow counts - was it intentional and is it indeed expected?